shop/dump.sql

```diff
@@ -528,7 +528,7 @@ CREATE TABLE `ps_carrier` (
 
 LOCK TABLES `ps_carrier` WRITE;
 /*!40000 ALTER TABLE `ps_carrier` DISABLE KEYS */;
-INSERT INTO `ps_carrier` VALUES (1,1,0,'0','',1,0,0,0,0,1,0,0,'',0,0,0,0,0,0.000000,0),(2,2,0,'My carrier','',1,0,1,0,0,0,0,0,'',0,1,0,0,0,0.000000,0),(3,3,0,'My cheap carrier','',0,0,1,0,0,0,0,0,'',2,2,0,0,0,0.000000,0),(4,4,0,'My light carrier','',0,0,1,0,0,0,0,0,'',1,3,0,0,0,0.000000,0);
+INSERT INTO `ps_carrier` VALUES (1,1,0,'PrestaShop','',1,1,0,0,0,1,0,0,'',1,0,0,0,0,0.000000,0),(2,2,0,'My carrier','',1,1,1,0,0,0,0,0,'',1,1,0,0,0,0.000000,0),(3,3,0,'My cheap carrier','',0,1,1,0,0,0,0,0,'',2,2,0,0,0,0.000000,0),(4,4,0,'My light carrier','',0,1,1,0,0,0,0,0,'',1,3,0,0,0,0.000000,0),(5,5,0,'Poczta Polska Przedpłata','',1,0,0,1,0,0,0,0,'',1,0,0,0,0,50.000000,0),(6,6,0,'Poczta Polska Pobranie','',1,1,0,1,0,0,0,0,'',1,1,0,0,0,0.000000,0),(7,6,0,'Poczta Polska Pobranie','',1,0,0,1,0,0,0,0,'',1,1,0,0,0,50.000000,0),(8,8,0,'Paczkomat InPost Przedpłata','',1,0,0,1,0,0,0,0,'',1,2,41,38,64,25.000000,0),(9,9,0,'Paczkomaty InPost Pobranie','',1,0,0,1,0,0,0,0,'',1,3,41,38,64,25.000000,0),(10,10,0,'Kurier InPost Przedpłata','',1,0,0,1,0,0,0,0,'',1,4,0,0,0,50.000000,0),(11,11,0,'Kurier InPost Pobranie','',1,0,0,1,0,0,0,0,'',1,5,0,0,0,50.000000,0),(12,12,0,'Połącz zamówienia','',1,0,0,1,0,1,0,0,'',1,6,0,0,0,50.000000,0),(13,13,0,'Odbiór osobisty','',1,0,1,1,0,1,0,0,'',1,7,0,0,0,50.000000,0),(14,14,0,'Shipping','',1,1,0,1,0,0,0,0,'',1,8,0,0,0,50.000000,0),(15,14,0,'Shipping','',1,0,0,1,0,0,0,0,'',1,8,0,0,0,50.000000,0);
```

Comment:
Everything looks great, but there seems to be a typo here: _Paczkomat InPost Przedpłata_ and _Paczkomat**y** InPost Pobranie_

Reply:
Thank you for finding that typo. Well done! <img src="https://media0.giphy.com/media/9HWPheLPma1vYrWMYY/giphy.gif"/>